ui/dashboardApp/layout/main/Sider/index.tsx

```diff
@@ -128,8 +124,17 @@ function Sider({
     }
   }, [defaultCollapsed])
 
+  const wrapperRef = useCallback((wrapper) => {
+    if (wrapper !== null) {
+      wrapper.addEventListener('transitionend', (e) => {
+        if (e.target !== wrapper || e.propertyName !== 'width') return
+        triggerResizeEvent()
+      })
+    }
+  }, [])
```

Comment:
First time to see this kind of usage, interesting, related link: https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node